src/library/Uno.Material/Styles/Application/Colors.xaml

```diff
@@ -132,6 +132,13 @@
 	<SolidColorBrush x:Key="OnSurfaceSelectedBrush"
 					 Color="{StaticResource OnSurfaceColor}"
 					 Opacity="0.08"/>
+	
+	<!-- Translucent Colors -->
```

Comment:
Do we really need those brushes? Let's at least use names from the color palette, not color01 :D

Reply:
@MatFillion  Problem is that color exists in the TextBox style but not the brushes. What do we do ?

Reply:
Let's keep it in TextBox.xaml then, use the name from the Variables in zeplin.

Reply:
Done. Will add remaining TextBox colors and brushes in another PR.